org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/adapters/HoverTypeAdapter.xtend

```diff
@@ -37,15 +38,17 @@ class HoverTypeAdapter {
 	protected def readContents(JsonReader in) throws IOException {
 		val nextToken = in.peek()
 		if (nextToken == JsonToken.STRING) {
-			val List<Either<String, MarkedString>> value = newArrayList(Either.forLeft(in.nextString))
+			val List<Either<String, MarkedString>> value = new ArrayList<Either<String, MarkedString>>()
```

Comment:
replaced it with a bit more verbose pure java

Reply:
If verbosity is a concern, you could remove the type arguments from the constructor invocation. They would be inferred. But I doubt it matters much.